Nanotrasen Mimir-Class Rehabilitation (Prison) Vessel

Latentish opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds the Mimir-Class Rehabilitation (Prison) Vessel
Removes the Tartarus
It's meant to simulate perma and the gulag, mixed with a bunch of maint stuff and low security staff. Starts with 3 security and 1 patient/prisoner.
![2023 04 20-14 53 43](https://user-images.githubusercontent.com/55075747/233496215-50d2bdae-8515-41d6-8be1-ab7ceaf8562e.png)
![2023 04 20-14 53 46](https://user-images.githubusercontent.com/55075747/233496220-d6708625-1bea-433b-a8b6-d7b7228ebf47.png)
![mimirreal](https://user-images.githubusercontent.com/55075747/233496222-6ae72eb0-9e5d-4604-8624-e5f10f51030a.png)



<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
The tartarus was our security/perma ship, so I want something better.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Adds a mimir (sleepy)
code: adds some supporting code for the mimir (sleepy)
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Latentish committed:
Adds the Mimir dmm

titular

Latentish committed:
A mimir

Adds more map

Latentish committed:
Adds more stuff to mimir

Latentish committed:
more mimir

also adds an empty warden locker

Latentish committed:
Adds the air recycler and stuff

added an easy to use air recycler. turn it on 4head

Latentish committed:
Update nanotrasen_mimir.dmm

Erikafox commented:
please bundle the tart's removal in with this, afaik it's still purchasable.

Latentish committed:
More Mimir shenanigans

-adds areas
-adds electricity!!!!
-adds atmos
-hopefully sets the smelter up correctly

Latentish committed:
Makes the ship real

-Adds atmos
-Finishes power
-adds some decal work
-populates maint
-adds security access doors
-adds secrets to find under the floorboards
-Adds the json

Latentish committed:
Merge remote-tracking branch 'upstream/master' into Mimir-Class

Latentish committed:
attaches scrubber

ZephyrTFA commented on code/game/MapData/shuttles/nanotrasen_mimir.dm:
```diff
@@ -0,0 +1,39 @@
+//toilet =)
+
+/obj/structure/toilet/secret/mimir
+	secret_type = "/obj/item/storage/backpack/satchel/flat/mimir_transfer"
```
This probably isn't meant to be a text path

Latentish replied:
fixod

Latentish committed:
hopefully fixes problemos

Latentish committed:
Decalling

-Adds a lot of decals
-removes the paper airplane book

Latentish committed:
Adds some notes

-Adds papers hinting at secrets

Latentish committed:
Merge branch 'shiptest-ss13:master' into Mimir-Class

Latentish committed:
removes the tartar

-Removes the tartarus and its json

Latentish committed:
Small changes for the kitchen

-Adds a rolling pin and some small foodstuffs

Latentish committed:
no more spamton

-kromer

Latentish committed:
HAHAHA

connects the prechargers and engines

Latentish committed:
replaces the warden armor vest

The /vest/security/warden and /warden/alt vests have no sprites. This replaces them with the standard /nt version we're all familiar with until this is fixed.

Erikafox commented:
![dreamseeker_eNk6mP5PJh](https://user-images.githubusercontent.com/94164348/214623650-95958f1f-28b1-49dc-b902-fbb330340d51.png)
spellcheck
![dreamseeker_VtA3Z5Om2Z](https://user-images.githubusercontent.com/94164348/214623685-d8feebc7-5d50-4b60-b90c-b952f30f4a59.png)
you can pixelshift these, I would, having it in the middle feels off
![dreamseeker_PSlxFmOmtl](https://user-images.githubusercontent.com/94164348/214623893-4efc6b91-cae8-4e32-b39a-61fdcf3850d8.png)
ditto
![image](https://user-images.githubusercontent.com/94164348/214624174-7fa93356-a4ee-4331-bf87-97d441af2b43.png)
decal layering issue on the stand clear, not sure if I really like the orange techmaint, it takes away from the natural maint look.
![dreamseeker_3x26CwJmRK](https://user-images.githubusercontent.com/94164348/214624453-aab5e87f-adfd-4f3c-b450-d23b08a48926.png)
I'd take the techmaint out or make it run with the railings here
![dreamseeker_vxqOFseAjt](https://user-images.githubusercontent.com/94164348/214624558-3a2f452f-73b0-4522-94c9-b482abda45bd.png)
![td0vOwXa7F](https://user-images.githubusercontent.com/94164348/214626623-4bb6cc22-44b9-4c84-9070-a26fef57d6a6.png)
wrong path
![dreamseeker_XRl03TcZma](https://user-images.githubusercontent.com/94164348/214629990-08f11ada-ec98-41fc-8d60-a01c647170e3.png)
I would move the APC - Air alarm to share spare with the vending machines, then move the lockers over.
![dreamseeker_PZJarYY3KQ](https://user-images.githubusercontent.com/94164348/214630326-29f2251e-c00c-4bf2-9b03-1a491bd8d37b.png)
window shutters please

More review later.

Erikafox commented on _maps/configs/nanotrasen_mimir.json:
```diff
@@ -0,0 +1,29 @@
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"prefix": "NTSV",
+	"namelists": ["GENERAL", "NATURAL_AGGRESSIVE", "BEASTS"],
+	"map_name": "Nanotrasen Mimir-Class Rehabillitation Vessel",
```
Unless the Nanotrasen is intentional, I'd cut it.

Erikafox commented on _maps/configs/nanotrasen_mimir.json:
```diff
@@ -0,0 +1,29 @@
+		},
+		"Prisoner": {
+			"outfit": "/datum/outfit/job/prisoner",
+			"slots": 5
```
you talked about changing the rest of these numbers up, leaving a comment as a reminder.

Erikafox commented on _maps/configs/nanotrasen_mimir.json:
```diff
@@ -0,0 +1,29 @@
+			"slots": 5
+		}
+	},
+	"enabled": true
```
depending on how this runs, I may prefer this ship be adminspawn.

Erikafox reviewed (commented):
not a comprehensive, just a few things while I'm thinking

Zevotech commented:
5 (or more) prisoners can easily end up overpowering the sec staff if allowed any kind of freedom, especially in accessing ruins and their loot. Might need to lower it so that having a lot of prisoners is the ship owners choice when adding slots.

Latentish commented:
> 5 (or more) prisoners can easily end up overpowering the sec staff if allowed any kind of freedom, especially in accessing ruins and their loot. Might need to lower it so that having a lot of prisoners is the ship owners choice when adding slots.
I like this idea a lot.

Erikafox reviewed (commented):
![image](https://user-images.githubusercontent.com/94164348/215005086-0dc71c1f-df77-4742-a8d7-f78f7ed17bcd.png)
I would undo the silly wiring here. It doesn't really mesh with the ordered techmaint.
![StrongDMM_EhUd7JNh6P](https://user-images.githubusercontent.com/94164348/215005217-29d88f3b-b6fa-4428-8cbe-f850fa5f853a.png)
I'd get some wireknots under the windows to the prison.
![StrongDMM_4owwX2qjQ8](https://user-images.githubusercontent.com/94164348/215005368-68d0dd22-883d-4bde-bf10-ae265f426ce0.png)
little wiring nitpick, I'd use a 90 degree turn instead of the diag, since there's something on the diag.
![StrongDMM_oBnNQNsICb](https://user-images.githubusercontent.com/94164348/215005410-11733cca-28c1-4827-a92a-4a4cf057d2bc.png)
I'd pixelshift the button to the edge of the wall, or maybe even put it on the outer edge (top of the image) of the hull. 
Also not huge on having to climb railings to screw with prechargers, try a windoor in the middle maybe?
If you are insistent on keeping the orange techmaint pattern, I would like the tiles the railing is on to be plating. 
![StrongDMM_2Pk5jcrITT](https://user-images.githubusercontent.com/94164348/215005656-6d889ecb-31b0-4079-b648-4823bbddcf63.png)
Kinda like such.
![StrongDMM_V3QcqoiWAr](https://user-images.githubusercontent.com/94164348/215005737-5aca0f42-f016-441d-bfe3-222bedbca3b3.png)
This space is too cramped for what it does, they also need an autolathe. 
Ditch the holofan projector too imo.
![StrongDMM_ZzrOHjD2tw](https://user-images.githubusercontent.com/94164348/215005844-4bad50f5-bbe0-4c30-885b-96a719251dc8.png)
Give the bridge door an access lock, even if it's a gameplay concession I don't think it really makes sense.
![StrongDMM_tU9UhIwmLo](https://user-images.githubusercontent.com/94164348/215005944-2192e819-72ef-48e5-8aff-b90d58383d71.png)
I'm not saying you should subtype a chemsprayer to start filled with pepperspray and give it to the sec team. But that would be incredibly funny. Also move the tear gas and awards to the warden's box. 
![StrongDMM_PQEOGDaJ4P](https://user-images.githubusercontent.com/94164348/215006068-de0f8817-7b1e-43b9-9341-2ab446487d8c.png)
Move the stimpacks to sec, leave a note saying they're for the prisoners when they're out mining, give them standard emergency oxygen tanks instead of the double yellows. I don't think I've ever run out of air on a double yellow and I've played entire rounds in a gas mask
![StrongDMM_BbyZL293Dg](https://user-images.githubusercontent.com/94164348/215006189-8dbeb8dd-212e-45c6-bf3f-f428d5d2b000.png)
Move the AAC down a few pixels, put a second vent inside so it cycles faster. 
Put some rusted reinforced walls around, there's interior rust but not enough
![StrongDMM_jjDwQVtjlP](https://user-images.githubusercontent.com/94164348/215006341-f6b06f60-3996-4f15-b455-9918a14c3758.png)
This is so fucking illegal. At least use a falsewall that's connected the room. 
![StrongDMM_Gq6kviaTr2](https://user-images.githubusercontent.com/94164348/215006449-4dd7400a-8715-425e-b467-95c699f2c9a0.png)
Give em some athletic shorts too
![StrongDMM_2FOrEpkzjq](https://user-images.githubusercontent.com/94164348/215006497-bc663272-bca2-4465-9895-abe455c67bb7.png)
FYI you can give this dood some custom lines via var editing. 
![StrongDMM_L6i9ZeFDUo](https://user-images.githubusercontent.com/94164348/215006536-94609896-781c-412d-9c3f-ddae097e9b46.png)
layering
![StrongDMM_ftOLVEuC6u](https://user-images.githubusercontent.com/94164348/215006555-3b64776b-214e-4286-af2b-189c519b0038.png)
try plating, or grey techmaint under conveyors. 
![StrongDMM_PhXbMR0Lpw](https://user-images.githubusercontent.com/94164348/215006605-5d04b87f-17eb-4b6f-9a83-a75e88d3896b.png)
I'm actually begging you to not use diag wires under false walls
The Central Hallway and Brig Cells need to have at least another scrubber + vent. I'd maybe do the same in Dormitory 3
![StrongDMM_exK9jxINlR](https://user-images.githubusercontent.com/94164348/215006787-87d28de3-e7d5-4712-a0b1-403d4bf1634e.png)
See this?
![StrongDMM_gmAn1nhOQ6](https://user-images.githubusercontent.com/94164348/215006875-48271fe1-fb78-4221-acb0-c52ef548cc16.png)
Try this instead.
![StrongDMM_mOILKsqHQh](https://user-images.githubusercontent.com/94164348/215006952-44ff9dc6-131d-4ffc-8e29-81bde39a5a47.png)
these'll layer weird, put the directional windows on the outside of the tiles instead.
![StrongDMM_2O1KUBvoZW](https://user-images.githubusercontent.com/94164348/215007012-3bbb274f-cb95-4f2f-8647-f918e6bd11c1.png)

Latentish committed:
mimir patch 1

- reduces prisoner count to 2
- nerfs prisoner meds substantially
- redoes some wiring and adds nodes under glass

Latentish commented:
guh guh guh me want see round but not time also not respond to advice yet ooga booga

Latentish commented:
I'm gonna draft this for now

Latentish committed:
Merge remote-tracking branch 'upstream/master' into Mimir-Class

Latentish committed:
Requested changes

-Fixes the smelter
-Makes the smes area bigger
- Changes some maint plating
- moves switches, buttons, and consoles
- adds a note about atmos
- changes the "prisoner" role to "patient"

Latentish commented:
Last commit also made the bridge door sec locked.

Latentish commented:
And adds an autolathe.

Latentish committed:
minor change

-Replaces funny implant radios with real radios
-removes the stand clear decal in the smelter
-fixes aneast

Erikafox committed:
Merge branch 'master' into Mimir-Class

Erikafox committed:
Merge branch 'master' into Mimir-Class

Erikafox commented:
The ship played fine last time I saw it but I'd still like to talk about making it a-spawn

Daved27hundred commented:
needs another pacman because with the recent change to how ships are piloted this thing guzzles power like crazy, and one pacman isn't enough.

Zevotech commented:
replace the rolling table in med because of the bug that makes it leave things behind. Lost the surgery bag once to that, wasn't fun.

Latentish committed:
makes mimir aspawn

Yeahhhhh this should probably be aspawn only.

Latentish committed:
Merge branch 'Mimir-Class' of https://github.com/Latentish/Shiptest into Mimir-Class

Erikafox commented:
![image](https://user-images.githubusercontent.com/94164348/223859157-b92f7e65-b7f9-4c6a-8d1d-935b155ffc2b.png)
![image](https://user-images.githubusercontent.com/94164348/223859179-c9646e17-f957-4d87-8925-5acb1147b1bd.png)
window shutters on both exterior windows
![image](https://user-images.githubusercontent.com/94164348/223859307-6659144d-13f5-4334-9c49-09391651fbae.png)
make the pump to the air recycler, and to one of the airtanks start on.
![firefox_2UFi3DA6jA](https://user-images.githubusercontent.com/94164348/223859444-267ceec1-3d4c-4a30-80db-4e42c4624bed.png)
change the x on these to -28

Latentish committed:
Merge remote-tracking branch 'upstream/master' into Mimir-Class

Latentish committed:
Based and compliance pilled

-Replaces rolling table (they are sgt coded)
-Adds window shutters
-Makes the air recycler pump on.
-Makes a gas tank pump on.
-Replaces credits with a singular money bag.

Latentish committed:
Forced naming schemes eat my

-Makes the shutters properly named.

Latentish committed:
Minor changes

-Removes "Scum" as a prisoner ID default registered name.
- Adds shutters to the mining bay and airlock.
- Makes the Mimir Player Spawnable

Latentish committed:
Remaps the helm and security dorms

Titular

Erikafox committed:
Merge branch 'master' into Mimir-Class

Latentish committed:
Small fix

- wires the holofield gens
- Fixes some fencing =)

Erikafox reviewed (changes requested):
This needs a stationary docking port on the exterior. Ping me in ruinyard or look at the Borea for an example

Latentish committed:
The docking port in question

-Adds the requested docking port.
